perfkitbenchmarker/gcp/gce_virtual_machine.py

```diff
@@ -223,9 +226,18 @@ def AddMetadata(self, **kwargs):
     vm_util.IssueCommand(cmd)
 
 
-class ContainerizedGceVirtualMachine(GceVirtualMachine,
-                                     linux_vm.ContainerizedDebianMixin):
+class DebianContainerizedGceVirtualMachine(GceVirtualMachine,
```

Comment:
I talked to amey about this a lot and we believe this is the only way to achieve this here. Also Containerized has nothing Debian specific about it, and Debian has nothing Containerized specific about it.